trino/trino.go

```diff
@@ -649,7 +650,7 @@ func (st *driverStmt) exec(ctx context.Context, args []driver.NamedValue) (*stmt
 			if err != nil {
 				return nil, err
 			}
-			if arg.Name == trinoUserHeader {
+			if strings.HasPrefix(arg.Name, trinoHeaderPrefix) {
```

Comment:
It does not look right. We are still adding `trinoUserHeader` always below.

Reply:
very good catch, thanks @losipiuk